cmd/broker/ingress/main.go

```diff
@@ -108,11 +125,20 @@ func getRequiredEnv(envKey string) string {
 	return val
 }
 
+func asBool(envVal string) bool {
```

Comment:
if you use the above mentioned kelsey env config stuff, you won't need this.

Reply:
Removed, thanks!